build.gradle.kts

```
@@ -35,3 +35,9 @@ allprojects {
         }
     }
 }
+
+// Workaround for https://youtrack.jetbrains.com/issue/KT-49109 that allows building on Apple Silicon
```

Change suggestion in an attempt to trigger GitHub checks to run.

```suggestion
// Workaround for https://youtrack.jetbrains.com/issue/KT-49109 that allows building on Apple Silicon.
```